src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -163,6 +163,7 @@ def load_arguments(self, _):
     for scope in ['disk create', 'snapshot create']:
         with self.argument_context(scope) as c:
             c.argument('source', help='source to create the disk/snapshot from, including unmanaged blob uri, managed disk id or name, or snapshot id or name')
+            c.argument('secure_vm_disk_encryption_set', options_list=['--securevm-disk-encryption-set'], help='Name or ID of disk encryption set created with ConfidentialVmEncryptedWithCustomerKey encryption type.')
```

Comment:
We plan to change all `--securevm-xxx` to `--secure-vm-xxx` for parameter names
https://github.com/Azure/azure-cli/issues/22199#issuecomment-1156496647
```suggestion
            c.argument('secure_vm_disk_encryption_set', help='Name or ID of disk encryption set created with ConfidentialVmEncryptedWithCustomerKey encryption type.')
```